frontend/src/components/RequestFeeTooltip.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Tooltip class="-mr-3" show-outside-of-closest-reference-element>
+    <div class="h-full flex flex-col justify-center" data-test="tooltip-trigger">
+      <img class="h-6 w-6 cursor-help" src="@/assets/images/help.svg" />
+    </div>
+    <template #hint>
+      <div data-test="tooltip-text">
+        The fee amount is composed of two parts:<br />
+        <ul class="pl-5">
+          <li>• the liquidity provider fee</li>
+          <li>• the protocol fee</li>
+        </ul>
+        <br />
+        The liquidity provider fee is paid out to the liquidity provider servicing the request,
+        while the protocol fee stays with the contract and supports the Beamer platform
+        development.<br />
+        Note that the fee is paid on top of the token amount being transferred,so that the token
+        amount received on the target rollup is exactly the same as the token amount sent from the
+        source rollup.
+        <span v-if="formattedMinFee" data-test="formatted-minimum-fee">
+          <br /><br />
+          The minimum fee for a transfer is
+          <span class="font-bold">{{ formattedMinFee }}</span>
+        </span>
+      </div>
+    </template>
+  </Tooltip>
+</template>
+
+<script lang="ts" setup>
+import { toRefs } from 'vue';
+
+import Tooltip from '@/components/layout/Tooltip.vue';
+import { useMinRequestFee } from '@/composables/useMinRequestFee';
+import type { Chain, Token } from '@/types/data';
+
+interface Props {
+  chain: Chain | undefined;
+  token: Token | undefined;
+}
```

Comment:
I'm thinking if it would make sense to call the `useMinRequestFee` composable in the `RequestSourceInputs` component and just pass the `formattedMinFee` as a prop. This would make the `RequestFeeTooltip` a purely visual component with no logic at all. We were trying to achieve this distinction in the past, see the `TransferSummary` for example.

Reply:
Done :heavy_check_mark: